tests/scripts/ncp_mode

```diff
@@ -238,6 +242,13 @@ test_teardown()
     exit ${EXIT_CODE}
 }
 
+restart_avahi_daemon()
+{
+    # Restart the avahi-daemon on the host, not the mdns service in docker.
```

Comment:
`not the mdns service in docker` What does `mdns` mean here? If you meant `avahi-daemon` probably it's better to remove `mdns` here because it could get confused with the `mdns` daemon.